.gitattributes

```diff
@@ -0,0 +1,8 @@
+/driver-testsuite export-ignore
```

Comment:
Yes and the alternative is using --prefer-source in the other drivers travis setup.